### PR TITLE
Add changelog for v2.27.x release

### PR DESCRIPTION
**Pull request by tirumerla**
- Add changelog for 2.27.x release

### CODE REVIEW

File: docs/modules/ROOT/pages/changelog.adoc [hunk 1]
```diff
@@ -1,5 +1,21 @@
 = Changelog
 
+[[release-2024-09-05]]
+== September 5th, 2024
+
+* *General*
+** Feature: UI redesign and improvements for the Defender 2.0 dashboard.
+** Feature: Add support for the proposals pagination.
+** Feature: Add support for EIP 1559 for fuse network.
```
Comment (cairoeth) [inline docs/modules/ROOT/pages/changelog.adoc:9]:
```suggestion
** Feature: Add support for EIP 1559 for Fuse network.
```

File: docs/modules/ROOT/pages/changelog.adoc [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+** Feature: Improve resiliency of the relayers by retrying transactions with specific error responses from providers.
```
Comment (cairoeth) [inline docs/modules/ROOT/pages/changelog.adoc:10]: specific to relayers?

File: docs/modules/ROOT/pages/changelog.adoc [hunk 3]
```diff
@@ -1,5 +1,21 @@
...
+** Feature: Allow EOAs to be used for upgrade approval process.
```
Comment (cairoeth) [inline docs/modules/ROOT/pages/changelog.adoc:11]: specific to deploy?

File: docs/modules/ROOT/pages/changelog.adoc [hunk 4]
```diff
@@ -1,5 +1,21 @@
...
+** Refactor: Change MATIC token symbol to POL token according to changes on polygon network.
```
Comment (cairoeth) [inline docs/modules/ROOT/pages/changelog.adoc:12]:
```suggestion
** Refactor: Change MATIC token symbol to POL token according to changes on Polygon network.
```

File: docs/modules/ROOT/pages/changelog.adoc (hunk 2 above)
Comment (tirumerla) [reply docs/modules/ROOT/pages/changelog.adoc:10]: Just the 9 & 10 others are not

File: docs/modules/ROOT/pages/changelog.adoc (hunk 3 above)
Comment (tirumerla) [reply docs/modules/ROOT/pages/changelog.adoc:11]: I'm combining everything under general, which we have been doing from past few releases...we can change it later if we see the need

**Review by cairoeth — APPROVED** [review]

**Review by CoveMB — APPROVED** [review]

**Pull request merged**